pkg/rehearse/jobs.go

```diff
@@ -192,18 +200,68 @@ func ExecuteJobs(toBeRehearsed map[string][]prowconfig.Presubmit, prNumber int,
 		}
 	}
 
-	if len(rehearsals) > 0 {
-		for _, job := range rehearsals {
-			created, err := submitRehearsal(job, refs, logger, pjclient)
-			if err != nil {
-				logger.WithError(err).Warn("Failed to execute a rehearsal presubmit")
-			} else {
-				logger.WithFields(pjutil.ProwJobFields(created)).Info("Submitted rehearsal prowjob")
-			}
-		}
-	} else {
+	if len(rehearsals) == 0 {
 		logger.Warn("No job rehearsals")
```

Comment:
Not a warning really

Reply:
That has been in there from the beginning. We have extra output now that makes this unnecessary. Removed.